kardinal_version/kardinal_version.go

```diff
@@ -0,0 +1,5 @@
+package kardinal_version
+
+const (
+	KardinalVersion = "0.4.3"
```

Comment:
the version will get written and read from `kurtosis_version.txt` and injected here using `-X` ldflag during nix build